packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -30,6 +30,7 @@
             android:label="@string/title_activity_main"
             android:theme="@style/AppTheme.NoActionBarLaunch"
             android:launchMode="singleTask"
+            android:screenOrientation="portrait"
```

Comment:
On Android Studio there is a warning aka yellow underline indicating that we should be aware to allow the user to use a horizontal version, but since we don't need it by now maybe we could add next line behinde to prevent the warning:
`tools:ignore="LockedOrientationActivity"`